internal/commands/ip_access/update_inputs.go

```diff
@@ -0,0 +1,37 @@
+package ipaccess
+
+import (
+	"fmt"
+
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cli/user"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+)
+
+type updateInputs struct {
+	cli.ProjectInputs
+	IPAddress    string
+	NewIPAddress string
+	Comment      string
+}
+
+func (i *updateInputs) Resolve(profile *user.Profile, ui terminal.UI) error {
+	if err := i.ProjectInputs.Resolve(ui, profile.WorkingDirectory, false); err != nil {
+		return err
+	}
+	return nil
```

Comment:
if both `NewIPAddress` and `Comment` are blank, I'm guessing the update fails (or at least is a no-op).  do you want to catch that case here, potentially prompt for any necessary info or just return an error?